### PR TITLE
tweak ft.info and docs

### PR DESCRIPTION
**Pull request by ashtul**

### CODE REVIEW

**Comment by K-Jo** [comment]
@MeirShpilraien nudge

File: src/cursor.c [hunk 1]
```diff
@@ -290,33 +290,32 @@ int Cursor_Free(Cursor *cur) {
   return Cursors_Purge(cur->parent, cur->id);
 }
 
-void Cursors_RenderStats(CursorList *cl, const char *name, RedisModuleCtx *ctx) {
+int Cursors_RenderStats(CursorList *cl, const char *name, RedisModuleCtx *ctx) {
   CursorList_Lock(cl);
   CursorSpecInfo *info = findInfo(cl, name, NULL);
-  size_t n = 0;
+  if (!info) {
```
Comment (MeirShpilraien) [inline src/cursor.c:296]: @ashtul how can this even happen? each index must have a cursor list associated with it no?

File: docs/Commands.md [hunk 2]
```diff
@@ -301,51 +301,115 @@ Returns information and statistics on the index. Returned values include:
 * Number of distinct terms.
 * Average bytes per record.
 * Size and capacity of the index buffers.
+* Indexing state and percentage as well as failures
+
+Optional
+
+* Statistics about the `garbage collector`.
```
Comment (MeirShpilraien) [inline docs/Commands.md:308]: Should be mentioned its not appeared only on NOGC

**Review by MeirShpilraien — COMMENTED** [review]
Small comments/questions

File: src/cursor.c [hunk 3]
```diff
@@ -290,33 +290,32 @@ int Cursor_Free(Cursor *cur) {
   return Cursors_Purge(cur->parent, cur->id);
 }
 
-void Cursors_RenderStats(CursorList *cl, const char *name, RedisModuleCtx *ctx) {
+int Cursors_RenderStats(CursorList *cl, const char *name, RedisModuleCtx *ctx) {
   CursorList_Lock(cl);
   CursorSpecInfo *info = findInfo(cl, name, NULL);
-  size_t n = 0;
+  if (!info || info->used == 0) {
```
Comment (MeirShpilraien) [inline src/cursor.c:296]: @ashtul again I do not think it can be NULL, also why not printing it anyway? why change it and break the ft.info reply?

Comment (ashtul) [reply src/cursor.c:296]: We had `if(info)` before, where we wouldn't (or thought we wouldn't) print all stats.
I'll remove it so we just keep on printing everything as we did

**Review by rafie — APPROVED** [review]

**Pull request merged**